src/query/api/v1/handler/prometheus/remote/write.go

```diff
@@ -318,6 +321,7 @@ func (h *PromWriteHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	batchErr := h.write(r.Context(), req, opts)
+	processingTime.Stop()
```

Comment:
Should this timer stop be deferred? We have some early returns if there are errors and the processing time won't properly be recorded? 

Also, is this semi-duplicating the `writeBatchLatency` metric?

Reply:
lol didn't see that! thanks!